_dbt/docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13
+    restart: always
+    environment:
+      POSTGRES_DB: my_db
+      POSTGRES_USER: user
+      POSTGRES_PASSWORD: password
+    ports: 
+      - "5432:5432"
+    volumes:
+    - ./postgres-data:/var/lib/postgressql/data
+
+  dbt-modeling:
```

Comment:
Please add below line, this service fails for me on arm chip
`platform: linux/amd64`